README.rst

```diff
@@ -87,34 +89,36 @@ before running the install::
     # Windows:
     set FC=D:\path\to\my\fortran.exe
 
-To build and install execute::
+To build and install, execute::
 
     cd /path/to/slycot_src/
     python setup.py install
 
-You can also use conda to build and install Slycot from source::
+You can also use conda to build and install Slycot from source, but
+you'll have to choose the right recipe directory.
 
-    conda build conda-recipe
-    conda install --use-local slycot
+On Linux you can choose between conda-recipe-openblas and
+conda-recipe-mkl
 
-If you prefer to use the OpenBLAS library, a conda recipe is available in
-``conda-recipe-openblas``.
+On macOS you should use conda-recipe-apple.
 
-Additional tips for how to install Slycot from source can be found in the
-``.travis.yml`` (commands used for Travis CI) and conda-recipe/ (conda
-pre-requisites) both which are included in the source code repository.
+On Windows you can try either conda-recipe-openblas or
+conda-recipe-mkl; they allow for Windows builds, but we don't
```

Comment:
```suggestion
On Windows you can try either ``conda-recipe-openblas`` or
``conda-recipe-mkl``; they allow for Windows builds, but we don't
```

Reply:
Actually, I have no idea, whether the windows builds work at all. The `bld.bat` in `conda-recipe-openblas` is unchanged from when it was still ``conda-recipe`` and uses environment variables for Generic BLAS/LAPACK. conda-forge/slycot-feedstock also has the same setup.

@moorepants, @murrayrm, @repagh you are maintaining the feedstock. Care to comment?

Should we move the Windows Script to `conda-recipe-win/`? Anybody got Slycot with MKL (with conda provided libs?) set up in Windows?